.ycm_extra_conf.py

```diff
@@ -36,7 +36,7 @@
 # CHANGE THIS LIST OF FLAGS. YES, THIS IS THE DROID YOU HAVE BEEN LOOKING FOR.
 flags = [
     '-DNDEBUG',
-    '-DWITH_TBB',
+    '-DNMC_HAVE_TBB',
```

Comment:
We can still have hard errors for most features like CUDA etc, but there will be cases when this might not be the case.